RUNNING_LOCALLY.md

```diff
@@ -30,55 +30,82 @@ The command will create Docker containers that will:
+
 ## Create a new register
 
 To create a new register, you first need to create a new entry in the register register.
 
 Choose a primary key for your register. This will also be the name of your register. For example, foobar.
 
 Add the name to the field register using:
-`$ echo '[{"field":"foobar", "datatype": "string", "phase": "alpha", "cardinality": "1", "text": "test field"}]' | python3 ./scripts/json-to-rsf/json2rsf.py field | curl field.local.openregister.org:8081/load-rsf -H "Content-Type: application/uk-gov-rsf" --data-binary @- -u foo:bar`
+`$ echo '[{"field":"foobar", "datatype": "string", "phase": "alpha", "cardinality": "1", "text": "test field"}]' | python3 ./scripts/json-to-rsf/json2rsf.py user field | curl field.local.openregister.org:8081/load-rsf -H "Content-Type: application/uk-gov-rsf" --data-binary @- -u foo:bar`
 
 Then, you'll need to create an entry in the register register using:
-`$ echo '[{"phase":"alpha","registry":"government-digital-service","text":"A test register","fields":["foobar", "name","start-date","end-date"],"register":"foobar"}]' | python3 ./scripts/json-to-rsf/json2rsf.py register | curl register.local.openregister.org:8081/load-rsf -H "Content-Type: application/uk-gov-rsf" --data-binary @- -u foo:bar`
+`$ echo '[{"phase":"alpha","registry":"government-digital-service","text":"A test register","fields":["foobar","name","start-date","end-date"],"register":"foobar"}]' | python3 ./scripts/json-to-rsf/json2rsf.py user register | curl register.local.openregister.org:8081/load-rsf -H "Content-Type: application/uk-gov-rsf" --data-binary @- -u foo:bar`
 
 Check this is successful at `field.local.openregister.org:8081/records` and `register.local.openregister.org:8081/records`. You should see the name of your register, such as `foobar`, as the first record. 
 
-Now you can update your local register to use this.
+Now you can update your local register.
 
 ## Change the configuration
 
 First, you'll need to change the configuration.
 
 Go to the `config.docker.register.yaml` file.
 
-Change `register: school-eng` to the name of your register, for example `register: foobar`.
+Change `register: country` to the name of your register, for example `register: foobar`.
 
 Update the schema in the same way, for example: `schema: foobar`.
 
 Restart your docker container:
 `docker restart openregister-register`
 
-You should see your new empty register at `127.0.0.1:8080`.
+You should see your new undefined register at `127.0.0.1:8080`.
 
 ## Load data into your register
 
 You can now load your own data in the new register.
 
+First, you need to populate the register with its register definition and field definitions. These should be the same as those listed in the field and register registers.
+
+```
+$ echo '[{"name":"foobar"}]' | python3 ./scripts/json-to-rsf/json2rsf.py system name | curl 127.0.0.1:8080/load-rsf -H "Content-Type: application/uk-gov-rsf" --data-binary @- -u foo:bar
```

Comment:
This is quite a lot of commands to run. Perhaps consider chaining them with `&&` ?